src/operator/tensor/elemwise_unary_op_basic.cc

```diff
@@ -81,11 +81,9 @@ The storage type of ``relu`` output depends upon the input storage type:
 
    - relu(default) = default
    - relu(row_sparse) = row_sparse
+   - relu(csr) = csr
```

Comment:
Please add fluent methods to CSRNDArray documentation like what was did for RowSparseNDArray https://github.com/apache/incubator-mxnet/blob/master/docs/api/python/ndarray/sparse.md#trigonometric-functions 
And add tests in https://github.com/apache/incubator-mxnet/blob/master/tests/python/unittest/test_sparse_ndarray.py#L847

Reply:
Done